docs/settings/security-settings.asciidoc

```diff
@@ -239,3 +239,113 @@ The format is a string of `<count>[ms\|s\|m\|h\|d\|w\|M\|Y]` (e.g. '20m', '24h',
 ============
 
 |===
+
+[float]
+[[audit-logging-settings]]
+===== Audit logging settings
+
+You can enable audit logging to support compliance, accountability, and security. When enabled, {kib} will capture:
+
+- Who performed an action
+- What action was performed
+- When the action occurred
+
+For more details and a reference of audit events, refer to <<xpack-security-audit-logging>>.
+
+[cols="2*<"]
+|===
+| `xpack.security.audit.enabled`
+| Set to `true` to enable audit logging for security events. *Default:* `false`
+|===
+
+[float]
+[[audit-logging-settings-beta]]
+===== ECS audit logging settings
+
+beta[]
+
+To enable the <<xpack-security-ecs-audit-logging, ECS audit logger>>, specify where you want to write the audit events using `xpack.security.audit.appender`.
+
+[cols="2*<"]
+|===
+| `xpack.security.audit.appender`
+| experimental[] Optional. Specifies where audit logs should be written to and how they should be formatted.
```

Comment:
Sorry if there was already a conversation about this that I missed -- but we have the new settings described as "experimental" while the feature itself is "beta". Is there a reason for that discrepancy?

Reply:
Is there a difference between “beta” and “experimental”? Platform team suggested adding this in which made sense to me but if experimental is a stage before beta then that is probably not correct. Happy to remove the experimental flag

Reply:
> Is there a difference between “beta” and “experimental”? 

Great question! Generally speaking, "experimental" features are something that we're exploring, but may change significantly or be removed entirely.

"Beta" features are perhaps less polished, and may still change, but we're more confident that that they'll stick around and become GA in the near future.

The definitions from our docs:

> This functionality is in beta and is subject to change. The design and code is less mature than official GA features and is being provided as-is with no warranties. Beta features are not subject to the support SLA of official GA features.

> This functionality is experimental and may be changed or removed completely in a future release. Elastic will take a best effort approach to fix any issues, but experimental features are not subject to the support SLA of official GA features.

Reply:
Cool, thanks for clarifying - I think this should be beta then rather than experimental! Will remove from the docs